tests/functional/z_cluster/nodes/test_nodes_maintenance.py

```
@@ -610,3 +624,277 @@ def test_pdb_check_simultaneous_node_drains(
+                self._wait_for_stable_degraded(ct_pod)
+                self._run_write_io(pod_obj)
+                verify_data_integrity(pod_obj, "fio-rand-write", original_md5)
+
+            elif live_hosts >= k:
+                log.info(f"Tier 3: {k} <= {live_hosts} < {min_size}")
+                time.sleep(60)
+                write_ok = self._try_write_io(pod_obj)
+                if write_ok:
+                    log.warning(
+                        f"Write succeeded with {live_hosts} hosts "
+                        f"(below min_size={min_size}), may be transient"
```

Ruff (0.15.20)

[warning] Logging statement uses f-string

(G004)